showyourwork/workflow/build.smk

```diff
@@ -69,37 +68,29 @@ if (paths.user().temp / "config.json").exists():
     # Resolve ambiguities in rule order
     ruleorder: syw__compile > syw__arxiv
 
-
     # Include custom rules defined by the user
     include: (paths.user().repo / "Snakefile").as_posix()
-    process_user_rules()
 
+    process_user_rules()
 
     # Hack to display a custom message when a figure output is missing
     patch_snakemake_wait_for_files()
 
-
     # Snakemake workflows complete successfully if there's no rule to generate
     # a given file but it is present on disk. This is bad for third-party
     # reproducibility, so here we hack it to require all inputs to be present.
     if config["require_inputs"]:
         patch_snakemake_missing_input_leniency()
 
-
-else:
-
-
-    if run_type != "clean":
-        raise exceptions.MissingConfigFile()
+    pass
```

Comment:
After adding the new conditional at the top, this isn't still needed anymore, is it?

Reply:
I think you are correct.

Actually, could the entire if statement be removed now, since the check is at the top?